source/backend/dal/Repositories/AcquisitionFileRepository.cs

```diff
@@ -755,6 +755,15 @@ public PimsAcquisitionFile GetByAcquisitionFilePropertyId(long acquisitionFilePr
                 .FirstOrDefault(a => a.PimsPropertyAcquisitionFiles.Any(x => x.PropertyAcquisitionFileId == acquisitionFilePropertyId));
         }
 
+        public PimsProperty GetProperty(long acquisitionFilePropertyId)
+        {
+            return this.Context.PimsPropertyAcquisitionFiles.AsNoTracking()
+                .Include(p => p.Property)
+                .Where(p => p.PropertyAcquisitionFileId == acquisitionFilePropertyId)
+                .Select(p => p.Property)
```

Comment:
nit: it might make more sense to have the caller do this - to make this method more reusable.